### PR TITLE
ASSETS-30847 AEM Data Service - Stats API Documentation

### PR DESCRIPTION
**Pull request by sthyagarajan**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/api/experimental/stats/index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: Stats API
```
Comment (ireasor) [inline src/pages/api/experimental/stats/index.md:2]: This is a customer-facing page and link, so we should be formal here and call this something like Statistics.

**Review by ireasor — CHANGES_REQUESTED** [review]

File: src/pages/index.md [hunk 2]
```diff
@@ -76,6 +76,12 @@ A comprehensive content management solution for building websites, mobile apps a
 
 * [Workflow API](https://experienceleague.adobe.com/docs/experience-manager-65/developing/extending-aem/extending-workflows/workflows-program-interaction.html?lang=en)
 
+<DiscoverBlock slots="link, text"/>
+
+**Stats**
+
+* [Stats API (Experiemental)](./api/experimental/stats/index.md)
```
Comment (raducotescu) [inline src/pages/index.md:83]:
```suggestion
* [Statistics API (Experimental)](./api/experimental/stats/index.md)
```

**Review by raducotescu — CHANGES_REQUESTED** [review]
Please correct the typo outlined in my commit suggestion before merging.

File: src/pages/api/experimental/stats/index.md (hunk 1 above)
Comment (sthyagarajan) [reply src/pages/api/experimental/stats/index.md]: Updated it to Statistics

**Review by ireasor — APPROVED** [review]

**Pull request merged**